app/components/gh-members-lab-setting.hbs

```diff
@@ -171,32 +171,35 @@
             </div>
             <div class="w-100 w-50-l flex flex-column flex-row-ns">
                 <div class="w-100 w-50-ns mr3-ns">
-                    <GhFormGroup>
+                    <GhFormGroup @errors={{this.settings.errors}} @hasValidated={{settings.hasValidated}} @property="stripePlans">
                     <label class="fw6 f8">Monthly price</label>
 
                     <div class="flex items-center justify-center mt1 gh-input-group gh-labs-price-label">
                         <GhTextInput
                             @value={{readonly this.stripePlans.monthly.amount}}
                             @type="number"
-                            @input={{action "setStripePlan" "month"}}
+                            @input={{action (mut this._scratchStripeMonthlyAmount) value="target.value"}}
+                            @focus-out={{action "validateStripePlans"}}
                         />
                         <span class="gh-input-append"><span class="ttu">{{this.stripePlans.monthly.currency}}</span>/month</span>
                     </div>
                     </GhFormGroup>
                 </div>
                 <div class="w-100 w-50-ns ml2-ns">
-                    <GhFormGroup @class="description-container">
+                    <GhFormGroup @errors={{this.settings.errors}} @hasValidated={{settings.hasValidated}} @property="stripePlans">
                     <label class="fw6 f8">Yearly price</label>
                     <div class="flex items-center justify-center mt1 gh-input-group gh-labs-price-label">
                         <GhTextInput
                             @value={{readonly this.stripePlans.yearly.amount}}
                             @type="number"
-                            @input={{action "setStripePlan" "year"}}
+                            @input={{action (mut this._scratchStripeYearlyAmount) value="target.value"}}
+                            @focus-out={{action "validateStripePlans"}}
                         />
                         <span class="gh-input-append"><span class="ttu">{{this.stripePlans.yearly.currency}}</span>/year</span>
                     </div>
                     </GhFormGroup>
                 </div>
+                <GhErrorMessage @errors={{settings.errors}} @property="stripePlans" class="w-100 red"/>
             </div>
```

Comment:
This looks like it fixes it to me, but I'm not a frontend dev 😬 

```suggestion
            </div>
            <div class="w-100 w-50-l flex flex-column flex-row-ns">
                <GhErrorMessage @errors={{settings.errors}} @property="stripePlans" class="w-100 red"/>
            </div>
```

![image](https://user-images.githubusercontent.com/3798302/86384235-5cfdd500-bc86-11ea-949e-a4018d4b1b0c.png)

also
* nitpick: 1 is valid, but the error says "greater than 1"
* might be cool to add the currency in the error: "Please enter a value greater than $1.00"

pinging @JohnONolan for feedback